frontend/post/postDirective.js

```diff
@@ -378,10 +393,15 @@
         })();
     });
 
+    function getTemplateUrl() {
+        const MOBILE_SCREEN = screen.width <= 960;
```

Comment:
doesnt have a utils file? so you can put this definition there, and use everywhere in application?